src/components/Home/About/About.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useState } from 'react';
+import './about.css';
+
+const About = () => {
+    const [companies, setCompanies] = useState([
+        {name: "Huawei", link: "https://www.huawei.com/minisite/icpc2024/en/"},
+        {name: "Google", link: "https://codingcompetitionsonair.withgoogle.com/"},
+        {name: "Meta", link: "https://www.facebook.com/codingcompetitions/hacker-cup"},
+        {name: "Shopee", link: "https://careers.shopee.sg/codeleague/"},
+        {name: "Codegoda", link: "https://codegoda.io/"},
+        {name: "Acer", link: "https://news.acer.com/acer-sponsors-the-international-olympiad-in-informatics-2022"},
+        {name: "Traveloka", link: "https://competition.binus.ac.id/icpc2018/"},
+        {name: "Sirclo", link: "http://ngoding-seru.jonathanirvin.gs/"},
+    ]);
+    const [socials, setSocials] = useState([
+        {name: "Instagram", link: "https://www.instagram.com/binusjollybee/"},
+        {name: "Linkedin", link: "https://www.linkedin.com/company/jollybee-binus/"}
+    ]);
```

Comment:
It's better to put all these data constants inside a single file (or directory), i.e. `src/components/data.jsx`, so it is easier to find and modify them

Reply:
okay, done